marginalia.el

```diff
@@ -70,6 +70,36 @@ It can also be set to an integer value of 1 or larger to force an offset."
   "Use whitespace margin for window widths larger than this value."
   :type 'integer)
 
+(defcustom marginalia-file-age-function #'marginalia--file-age
+  "Function which genrates a file age string from the file time."
+  :type 'function)
+
+(defcustom margialia-max-relative-age (* 60 60 24 14)
```

Comment:
Please check for typos

Reply:
Gah. Typos.

Reply:
Fixed, and the rest has been checked again.